platform/keyword-registry/src/test/java/org/codice/ddf/keyword/KeywordRegistryImplTest.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.keyword;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.Matchers.hasSize;
+import static org.junit.Assert.assertThat;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Optional;
+import org.junit.Test;
+
+public class KeywordRegistryImplTest {
+
+  @Test
+  public void testGetUndefinedKeyword() {
+    KeywordRegistryImpl keywordRegistry = new KeywordRegistryImpl();
```

Comment:
I made `KeywordRegistryImpl` a member variable but left it up to each test to set the `keywordList`. Mainly because I think it makes each test clearer than it would if I declared a `keywordList` in `setup`